Doc/constraints.txt

```diff
@@ -7,18 +7,20 @@
 # Direct dependencies of Sphinx
 babel<3
 colorama<0.5
```

Comment:
I kept `colorama` at `<0.5` because packages that still use `0.X` distributions often sort-of tend to treat the `X` as their major version number. Sounds alright?

Reply:
Yeah, for example, SemVer says during 0.x anything goes.